libraryBuilds/com.fasterxml.jackson.core/com.fasterxml.jackson.core-jackson-databind/build.gradle

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2015 the authors of j2objc-common-libs-e2e-test
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+dependencies {
+    compile project(':com.fasterxml.jackson.core-jackson-core')
+    compile project(':com.fasterxml.jackson.core-jackson-annotations')
+    j2objcTranslation 'com.fasterxml.jackson.core:jackson-databind:2.5.3:sources'
+    testCompile 'junit:junit:4.11'
+}
+
+test {
+    testLogging {
+        // Provide full exception info on failure, instead
+        // of just pointing to an HTML file.
+        exceptionFormat = 'full'
```

Comment:
We use that on many build but it should be on all of them.

Use groovy style: `exceptionFormat 'full'`